src/input_cp2k_properties_dft.F

```diff
@@ -2137,10 +2102,12 @@ SUBROUTINE create_soc_section(section)
       CALL section_create(section, __LOCATION__, name="SOC", &
                           description="Switch on or off spin-orbit coupling. Use SOC "// &
                           "parameters from non-local pseudopotentials as given in "// &
-                          "Hartwigsen, Goedecker, Hutter, Eq.(18), (19) "// &
+                          "Hartwigsen, Goedecker, Hutter, Eq.(18), (19), "// &
                           "<https://doi.org/10.1103/PhysRevB.58.3641>, "// &
-                          "V^SOC_µν^(α) = ħ/2 < ϕ_µ | sum_ℓ ΔV_ℓ^SO(r,r') L^(α) | ϕ_ν >, "// &
-                          "α = x, y, z.", &
+                          "$V^\mathrm{SOC}_{\mu\nu}^{(\alpha)} = "// &
```

Comment:
There are two exponents in this term: `^\mathrm{SOC}` and `^{(\alpha)}`.
A simple way to check formulas is this page: https://www.mathjax.org/#demo